mozillaspeechlibrary/src/main/java/com/mozilla/speechlibrary/utils/ModelUtils.java

```diff
@@ -161,13 +159,13 @@ public static String getTFLiteFolder(@NonNull String modelRoot) {
     }
 
     @NonNull
-    public static String getLMFolder(@NonNull String modelRoot) {
-        return modelRoot + "/lm.binary";
+    public static String getAlphabetFolder(@NonNull String modelRoot) {
```

Comment:
alphabet is not needed and present anymore

Reply:
removed